### PR TITLE
blherg

### PR DESCRIPTION
**Pull request by chrismccord**

### CODE REVIEW

File: source/posts/2016-03-25-what-makes-phoenix-presence-special-sneak-peek.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+- has no single point of failure
+- has no single source of truth
+- relies entirely on the standard library with no operational
+dependencies
+- self heals
+
+Unlike most libraries and web frameworks in various languages, Phoenix
+does not require a central datastore to hold presence information.
+While having to deploy redis or similar datastores increases your
```
Comment (bcardarella) [inline source/posts/2016-03-25-what-makes-phoenix-presence-special-sneak-peek.md]: I think Redis is a proper name so it should be capitalized

File: source/posts/2016-03-25-what-makes-phoenix-presence-special-sneak-peek.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+through a central server. It also gives you resilience to failure
+because the system *automatically recovers from failures*. This
+includes other nodes going up or down or spotty networks causing
+netsplits and missed data replication.
+
+### Great platforms yield great solutions
+
+None of this would be possible without the innovations from Elixir and
+Erlang which gives us a distributed runtime that's unmatched by other
+languages. When we sat down to design Phoenix presence, instead of
```
Comment (bcardarella) [inline source/posts/2016-03-25-what-makes-phoenix-presence-special-sneak-peek.md]: Should "Presence" be capitalized here?

Comment (rsocci) [inline source/posts/2016-03-25-what-makes-phoenix-presence-special-sneak-peek.md]: capitalize` presence`

**Comment by courajs** [comment]
looks goooOooOood

**Pull request merged**